bluebottle/analytics/utils.py

```diff
@@ -0,0 +1,32 @@
+from importlib import import_module
+
+from django.conf import settings
+
+
+# NOTE: Stubbed method for testing
+def queue_analytics_record(timestamp, tags={}, fields={}):
+    # Get the default backend for analytics
+    def get_handler_class(handler): 
```

Comment:
Why is this function defined inline? Do we need a new one each time this function is called? Or does it need to be private?